web/profiles/webspark/webspark/config/install/pathauto.pattern.basic_page.yml

```diff
@@ -8,10 +8,10 @@ label: 'Basic Page'
 type: 'canonical_entities:node'
 pattern: '[node:menu-link:parents:join-path]/[node:title]'
 selection_criteria:
-  24e2ef1e-b842-4326-9666-87eef4bfd3c9:
+  a1eb5b64-2915-444e-a0cc-d5e6141f3102:
     id: 'entity_bundle:node'
     negate: false
-    uuid: 24e2ef1e-b842-4326-9666-87eef4bfd3c9
+    uuid: a1eb5b64-2915-444e-a0cc-d5e6141f3102
```

Comment:
Probably should avoid changing the UUID here on lines 11 and 14.

Reply:
This is because I ran the config update drush command and this is what was brought out of the database. I imagine that something changed which necessitated the update.